src/client/components/pages/collection.js

```diff
@@ -311,6 +317,7 @@ class CollectionPage extends React.Component {
 								bsSize="small"
 								bsStyle="danger"
 								className="margin-bottom-d5"
+								style={{marginBottom: '1rem', marginRight: '0.5rem'}}
```

Comment:
Let's remove the inline styles and change the `className="margin-bottom-d5"` above to `className="margin-bottom-1 margin-right-d5"